http/headers/feature-policy.json

```diff
@@ -1703,44 +1499,19 @@
               "edge": {
                 "version_added": false
               },
-              "firefox": [
-                {
-                  "version_added": "81",
-                  "partial_implementation": true,
-                  "notes": [
-                    "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
-                    "Firefox recognizes the <code>web-share</code> permissions policy, but this has no effect in versions of Firefox that do not support the <a href='https://developer.mozilla.org/docs/Web/API/Navigator/share'><code>share()</code></a> method."
-                  ]
-                },
-                {
-                  "version_added": "81",
-                  "notes": "Firefox recognizes the <code>web-share</code> permissions policy, but this has no effect in versions of Firefox that do not support the <a href='https://developer.mozilla.org/docs/Web/API/Navigator/share'><code>share()</code></a> method.",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.security.featurePolicy.header.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
-              "firefox_android": [
-                {
-                  "version_added": "81",
-                  "partial_implementation": true,
-                  "notes": "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements."
-                },
-                {
-                  "version_added": "81",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.security.featurePolicy.header.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
+              "firefox": {
+                "version_added": "81",
+                "partial_implementation": true,
+                "notes": [
+                  "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
+                  "Firefox recognizes the <code>web-share</code> permissions policy, but this has no effect in versions of Firefox that do not support the <a href='https://developer.mozilla.org/docs/Web/API/Navigator/share'><code>share()</code></a> method."
```

Comment:
For a header, no effect means no support, this note should be removed in favor of the correct version then.

Reply:
#15987 was opened to look further into these notes.  I agree that these notes imply `version_added: false`, but would not like to have them block this PR and would love to keep the scope of this PR as small as possible.